client/app/attend/attend.html

```diff
@@ -13,15 +13,54 @@
             </form>
         </div>
     </div>
-    <div class="row" ng-show='unverified'>
-        <div class="col-sm-12">
-            <div class='alert alert-info unverified'>
-                Your attendance needs to be verified by a mentor, this occurs randomly to make sure that nobody is dishonestly
-                logging attendance. Simply ask a mentor to verify your presence to log your attendance.<br/><br/>
-                If you do not log your attendance with a mentor, your unverified attendance will be recorded.
-            </div>
+    <div class="row">
+      <div class="col-sm-12" ng-show='unverified'>
+        <div class='alert alert-info unverified'>
+          Your attendance needs to be verified by a mentor, this occurs randomly to make sure that nobody is dishonestly
+          logging attendance. Simply ask a mentor to verify your presence to log your attendance.<br/><br/>
+          If you do not log your attendance with a mentor, your unverified attendance will be recorded.
         </div>
+      </div>
+      <div class="col-sm-8 col-sm-offset-2" ng-show='unverified'>
+        <hr />
+      </div>
+      <div class="col-sm-12">
+        <h3>Previous Attendance:</h3>
+      </div>
+      <div class="col-sm-12">
+        <table class="table table-hover">
+          <thead>
+            <tr>
+              <td ng-click="setSortOrder('datetime')">Date
```

Comment:
at some point we might want to break this into a directive so that the directive can worry about putting the arrow on the header value but for now this is fine. Just a lot of duplicate code